lib/ecto/query/api.ex

```diff
@@ -428,6 +428,32 @@ defmodule Ecto.Query.API do
   The only downside is that it will show up as a fragment when
   inspecting the Elixir query.  Other than that, it should be
   equivalent to a built-in Ecto query function.
+
+  ## Mutatinng the query string with escape!/1
```

Comment:
```suggestion
  ## Mutating the query string with escape!/1
```

Reply:
I wanted to convey a strong term that gives pause to anyone thinking of using escape!/1 because of the implications.  Its definitely not completely accurate as you say.  I'd like to hear some additional views if there are any about the best term to use here.